src/components/Home/Keshav/Week5/Counter/index.tsx

```diff
@@ -0,0 +1,35 @@
+import React from 'react';
+import { CounterWrapper } from './styles';
+import { useSelector, useDispatch } from 'react-redux';
+import { incrementCounter, decrementCounter } from '../redux/actions/CounterActions';
+import { RootState } from '../redux/actions/types';
+
+const Counter: React.FC = () => {
+  const count = useSelector((state: RootState) => state.count);
+  const dispatch = useDispatch();
+
+  const increment = () => {
+    dispatch(incrementCounter());
+  };
+
```

Comment:
since it is single line, you can use inline function here.

Reply:
It is the standard way, I have seen across various tutorials and websites and I think it's better to keep separate function for dispatching actions, so that same can be reused if we want it multiple times in a file.